src/tests/java/com/util/CodeUtilTest.java

```diff
@@ -0,0 +1,67 @@
+package com.util;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CodeUtilTest {
+
+    @Test
+    public void testEscape() {
+        Assert.assertEquals("%80%u9e00Ё%00A",
+                CodeUtil.escape("\u0080\u9e00\u0401\u0000A"));
+    }
+
+    @Test
+    public void testUnescape() {
+        Assert.assertEquals("fo\u0011ar", CodeUtil.unescape("fo%11ar"));
+    }
+
+    @Test
+    public void testDbEncode() {
+        Assert.assertNull(CodeUtil.dbEncode(null));
+        Assert.assertEquals("%25E2%2599%25A7", CodeUtil.dbEncode("♧"));
+    }
+
+    @Test
+    public void testDbDecode() {
+        Assert.assertNull(CodeUtil.dbDecode(null));
+        Assert.assertEquals("â\u0099¢", CodeUtil.dbDecode("%E2%99%A2"));
+    }
+
+    @Test
+    public void testToText() {
+        Assert.assertEquals("", CodeUtil.toText(null));
+        Assert.assertEquals("foo", CodeUtil.toText("foo"));
+    }
+
+    @Test
+    public void testToHtml() {
+        Assert.assertEquals("", CodeUtil.toHtml(null));
+        Assert.assertEquals("foo", CodeUtil.toHtml("foo"));
+    }
+
+    @Test
+    public void testToUnicode() {
+        Assert.assertEquals("1",
```

Comment:
I think lines 45 +46 can be joined and be < 80 chars

Reply:
Done